templates/admin/org/list.tmpl

```diff
@@ -15,7 +15,6 @@
 					<table class="ui very basic striped table">
 						<thead>
 							<tr>
-								<th>ID</th>
```

Comment:
A proper listing is a different story, but I would like to keep this information.